todo_app/data/trello_items.py

```diff
@@ -0,0 +1,100 @@
+import requests, os
+from todo_app.trello_config import Config
+
+def build_auth_query():
+    return {'key' : Config.TRELLO_API_KEY, 'token' : Config.TRELLO_API_TOKEN}
+
+BASE_URL= "https://api.trello.com/1/"
+BOARD_NAME= "Aaronboard"
+
+def get_trello_board_id():
+    boards = requests.get(f"{BASE_URL}members/me/boards", params=build_auth_query())
+    boards_json = boards.json()
+    boardexists = 0
+    for list in boards_json:
+        if list['name'] == BOARD_NAME:
+            boardexists = 1
+            boardid = list['id']
+    
+    if boardexists == 0:  
+        boardid = build_trello_board()
+        return boardid
+    else:
+        return boardid 
```

Comment:
Could be tidied slightly:
```suggestion
    for board in boards_json:
        if board['name'] == BOARD_NAME:
            return board['id']
      
    boardid = build_trello_board()
    return boardid
```